cli/sawtooth_cli/admin.py

```diff
@@ -22,15 +21,19 @@
 from sawtooth_cli.admin_command.utils import ensure_directory
 
 
-def do_admin(args):
+def do_admin(args, parser):
     data_dir = ensure_directory('data', '/var/lib/sawtooth')
 
+    if args.admin_cmd is None:
```

Comment:
Well, what I have done is to make the user experience better. Instead of getting an error about the argument "None" being not supported, it spits out help on the subcommand. 
I thought this was a good solution, and found similar examples during my research.